src/gui/optionsdialog.ui

```diff
@@ -3147,14 +3147,7 @@ Disable encryption: Only connect to peers without protocol encryption</string>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout_5">
-                 <item row="0" column="0">
```

Comment:
The changes in this file look messy. Please remake it manually to avoid unnecessary changes.

Reply:
I made a new commit at 59c3025013775268e4e66e3509bf7b33924879b5 to make the diff for this PR more readable manually.
Didn't notice that Qt Designer shifted them